targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs.ejs

```diff
@@ -29,7 +29,29 @@ namespace PlayFab
 
     public static class PlayFabSettings
     {
-        static PlayFabSettings() { }
+        static PlayFabSettings() 
+        {
+#if UNITY_GAMECORE || UNITY_GAMECORE_XBOXONE || UNITY_GAMECORE_SCARLETT || MICROSOFT_GAME_CORE
+            PlatformString = "GDK";
+#else
+        switch (Application.platform)
```

Comment:
What about other platforms?

Reply:
Other platforms like PS5, PS4, Switch, Android, Windows, and iOS should set by converting the Platform.Runtime enum to a string directly if none of the GDK platform macros are set.

Reply:
Oh I see now! I'd misread the default block 👍